source/slides/13_ansible.rst

```diff
@@ -0,0 +1,520 @@
+.. _13_ansible:
+
+Ansible
+=======
+
+What is Ansible?
+----------------
+
+* Started in 2012
+* More than 600 contributors
+* Orchestration Engine
+
+  * Configuration Management
+  * Application Deployment
+  * Continuous Delivery
+
+Ansible Commands
+----------------
+
+.. rst-class:: build
+
+``ansible``
+  Low level command to execute simple Ansible resources
+``ansible-playbook``
+  Tool to run Ansible Playbooks
+``ansible-pull``
+  Tool to pull remote Ansible configurations and run the playbooks
+``ansible-doc``
+  Tool to show documentation on Ansible modules
+``ansible-galaxy``
+  Tool to manage roles from Ansible Galaxy, a shared repository of Ansible roles
+  maintained by the community
+
+Ansible Components
+------------------
+
+.. rst-class:: build
+
+**Modules**
+  Python-driven code that actually does the work in Ansible and executed on the
+  remote host.
+**Inventory**
+  Defines how Ansible with interact with remote hosts and defines hosts, groups
+  of hosts in a logical manner.
```

Comment:
Is it adding too much detail to also mention that you define host and group variables in inventory?